jwql/tests/test_edb_interface.py

```diff
@@ -17,26 +17,70 @@
 
         pytest -s test_edb_interface.py
 """
-from astropy.time import Time
 import pytest
+from astropy.time import Time
+from astroquery.mast import Mast
+
+from jwql.edb.edb_interface import mnemonic_inventory, query_single_mnemonic
+from jwql.utils.utils import get_config
+
+
+@pytest.mark.xfail
+def test_get_mnemonic():
```

Comment:
This is marked as `xfail` because of the potential problems with the `config.json`, yes? I recently learned there's a way to specify which errors should be ignored by the `xfail`: https://docs.pytest.org/en/latest/skipping.html#raises-parameter

If you don't think it's too restrictive, it might be wise to restrict the `xfail` tests like so:
```python
@pytest.mark.xfail(raises=(RuntimeError, FileNotFoundError))
```
to ignore only the errors that arise with config file problems.

(This might be wise to expand across all of our xfail tests, eventually...)

Reply:
Yes to the question and thanks for the tip. I added the restrictions.